packages/subgraph/src/mappings/superfluidGovernance.ts

```diff
@@ -61,6 +78,19 @@ export function handleCFAv1LiquidationPeriodChanged(
     ev.isKeySet = event.params.isKeySet;
     ev.liquidationPeriod = event.params.liquidationPeriod;
     ev.save();
+
+    // Update cfav1 liquidation period for token or default governance configs
+    if (event.params.superToken.equals(ZERO_ADDRESS)) {
+        const defaultGovernanceConfig = getOrInitDefaultGovernanceConfig(event.block);
+        defaultGovernanceConfig.liquidationPeriod = event.params.liquidationPeriod;
+        defaultGovernanceConfig.save();
+    } else {
+        const token = Token.load(event.params.superToken.toHexString());
+        if (token) {
+            token.liquidationPeriod = event.params.liquidationPeriod;
+            token.save();
+        }
+    }
```

Comment:
(follow-up comment to "TokenGovernance") most of these code blocks would make more sense like this:
```ts
const governance = getOrInitGovernanceConfig(event.params.superToken.toHexString(), event.block);
governance.liquidationPeriod = event.params.liquidationPeriod;
governance.updatedAtTimestamp = event.block.timestamp;
governance.updatedAtBlock = event.block.number
governance.save();
```
Would not need conditionals in these situations. Also token-specific governance would have timestamps.